api/src/main/java/org/apache/iceberg/expressions/BoundAggregate.java

```diff
@@ -29,19 +29,51 @@ protected BoundAggregate(Operation op, BoundTerm<T> term) {
 
   @Override
   public C eval(StructLike struct) {
-    throw new UnsupportedOperationException(this.getClass().getName() + " does not implement eval");
+    throw new UnsupportedOperationException(
```

Comment:
just a formatting change here?

Reply:
I think this is reformatted by `spotlessApply`